fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/responses/ViewHelpers.java

```diff
@@ -399,7 +399,17 @@ public Resource rdfsClass() {
      * @return content-bearing node for the given subject
      */
     public static Node getContentNode(final Node subject) {
-        return subject == null ? null : createURI(subject.getURI().replace("/" + FCR_METADATA, ""));
+        return subject == null ? null : NodeFactory.createURI(subject.getURI().replace("/" + FCR_METADATA, ""));
```

Comment:
@acoburn, I chose to use `NodeFactory.createURI` as a direct replacement for the previous `Node_ANY.createURI` in order to minimize the change in logic as well as keep the `getContentNode` and local `createURI` methods decoupled.
But, as you observed, the functionality is the same either way.
If there is a good reason to do so, I would be fine with changing this to use the local `createURI`.